catalogs/catalog-messaging-kafka/src/main/java/com/datastrato/gravitino/catalog/kafka/KafkaCatalogOperations.java

```diff
@@ -41,15 +56,48 @@ public class KafkaCatalogOperations implements CatalogOperations, SupportsSchema
 
   private final CatalogEntity entity;
   private final EntityStore store;
+  private final IdGenerator idGenerator;
+  private final String DEFAULT_SCHEMA_NAME = "default";
+  @VisibleForTesting final NameIdentifier defaultSchemaIdent;
+  @VisibleForTesting Properties adminClientConfig;
 
-  public KafkaCatalogOperations(CatalogEntity entity) {
+  // For testing only.
```

Comment:
can you use `@VisibleForTesting` or `@TestOnly` ?

Reply:
done